app/helpers/cart_helper.rb

```diff
@@ -0,0 +1,30 @@
+module CartHelper
+  def get_name_product pro_id
+    product = Product.find_by(id: pro_id)
+    product&.name
+  end
+
+  def get_price_product pro_id
+    product = Product.find_by(id: pro_id)
+    product&.price
+  end
+
+  def get_category_product pro_id
+    product = Product.find_by id: pro_id
+    product&.category_name
+  end
+
+  def count_item
+    @count = @carts.sum(&:second)
```

Comment:
Có nhất thiết phải dùng thêm biến @count đâu nhỉ ?
Viết kiểu &:second cũng Ok. Mà nếu nó tương minh hơn thì viết kiểu này
```suggestion
    @carts.sum{|pro_id, qty| qty}
```
Để đọc vào kiểu cái @carts ấy lưu cái key là pro_id còn value là cái qty

Reply:
dạ em cảm ơn anh